src/newton.jl

```diff
@@ -96,8 +92,6 @@ function newton_{T}(df::AbstractDifferentiableMultivariateFunction,
         df.g!(x, fjac)
         g_calls += 1
 
-        g = fjac'*fvec
```

Comment:
Forgot that I added this. g is not used except in the catch block so move it in there to save a matvec multiplication